Mollier/SAM.Core.Mollier.UI/Modify/AddMollierPoints.cs

```diff
@@ -2,20 +2,15 @@
 using System.Windows.Forms.DataVisualization.Charting;
 using System.Collections.Generic;
 using System.Linq;
+using System.Drawing;
 
 namespace SAM.Core.Mollier.UI
 {
     public static partial class Modify
     {
-
-        public static Series AddMollierPoints(this Chart chart, IEnumerable<UIMollierPoint> uIMollierPoints, MollierControlSettings mollierControlSettings)
+        public static Series AddMollierPoints(this Chart chart, IEnumerable<UIMollierPoint> mollierPoints, MollierControlSettings mollierControlSettings)
```

Comment:
for public method use full class name if possible:
mollierPoints -> uIMollierPoints
Exception: interface where prefix I to be skipped
Do not mind naming private variables inside the methods